app/src/main/java/ch/nevis/exampleapp/common/settings/Settings.kt

```diff
@@ -0,0 +1,20 @@
+package ch.nevis.exampleapp.common.settings
+
+/**
+ * Interface declaration of application settings. It provides application level settings properties.
+ */
+interface Settings {
```

Comment:
AllowClass2Sensors flag is needed for proper biometric authenticator enrollment checking, so I introduced this interface and its implementation that provides only this flag at this moment.